pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2158,17 +2161,26 @@ type StorageScopeSpec struct {
 type Node struct {
 	// +optional
 	Name string `json:"name,omitempty"`
-	// +kubebuilder:pruning:PreserveUnknownFields
 	// +nullable
 	// +optional
-	Resources v1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
This could be a breaking API change and cause configuration loss for any users who use `nodes.resources`. @travisn do we need to have a deprecation notice, or should we just be adding `resourceClassName` here and not in a sub-element?

Reply:
With the current change I have pushed I have kept both the fields, the older one and a newer ResourceClassName field,
So will keep older files as it is, and if the ResourceClassName is set then will give preference to it,what are your thoughts on this?

Reply:
Yes, we will want to keep both and makes sense to give preference to the new resourceClassName if specified.